src/FrequencyMapAgent.cpp

```diff
@@ -39,17 +39,18 @@
 #include <utility>
 #include <algorithm>
 
-#include "contrib/json11/json11.hpp"
+#include "geopm/json11.hpp"
 
 #include "geopm.h"
 #include "geopm_hash.h"
 
 #include "Environment.hpp"
-#include "PlatformIO.hpp"
-#include "PlatformTopo.hpp"
+#include "PlatformIOProf.hpp"
```

Comment:
For agents that load as a plugin, this header should be installed?

Reply:
I am now installing it, but we can still refer to it without the geopm/ since it is located in the same directory as this file.